tests/acceptance/features/webUIManageUsersGroups/addUsers.feature

```diff
@@ -136,4 +136,17 @@ Feature: add users
     And the user follows the password set link received by "correct@owncloud" using the webUI
     And the user sets the password to "%regular%" using the webUI
     And the user logs in with username "guiusr1" and password "%regular%" using the webUI
-    Then the user should be redirected to a webUI page with the title "Files - %productname%"
\ No newline at end of file
+    Then the user should be redirected to a webUI page with the title "Files - %productname%"
+
+  @skip @issue-90
+  Scenario: check if the sender email address is valid
+    Given the administrator has changed the email of user "%admin%" to "admin@owncloud.com"
```

Comment:
We discovered that this is not needed and not correct.
"email from" is decided by the system mail_domain and mail_from_address settings. For testing, those are currently set in ``run.sh`` to make ``foobar@owncloud.com``